YANG/tapi-photonic-media@2018-08-31.yang

```diff
@@ -250,15 +250,11 @@ module tapi-photonic-media {
                 uses otsi-termination-config-pac;
                 description "none";
             }
-            container nmc-config {

```

Comment:
I am not sure if this container is not needed anymore. I wonder how you will specify the media-channel spectrum band if it is not througth this configuration.

Reply:
Agree - added the spectrum property to otsi-termination-config-pac as per commit below